js/layout-javascript/agenda-code.js

```diff
@@ -2320,6 +2326,8 @@ DynamicList.prototype.addDetailViewData = function (entry) {
     // Define content
     if (dynamicDataObj.customFieldEnabled) {
       content = new Handlebars.SafeString(Handlebars.compile(dynamicDataObj.customField)(entry.originalData));
+    } else if (dynamicDataObj.type === 'file') {
```

Comment:
Why is this line in the agenda layout code but not in the other layouts?

Reply:
@tonytlwu Because agenda has different logic of defining content. Other layouts already have similar logic where input type file is processed.